pkgs/top-level/all-packages.nix

```diff
@@ -6113,7 +6113,7 @@ with pkgs;
 
   elmPackages = recurseIntoAttrs (callPackage ../development/compilers/elm { });
 
-  adobe_flex_sdk = callPackage ../development/compilers/adobe-flex-sdk { };
```

Comment:
You could add an alias for the old name : `adobe_flex_sdk = apache-flex-sdk;` to `pkgs/top-level/aliases.nix` to avoid breaking existing installations.

Reply:
Done!